doc/Security.xml

```diff
@@ -4882,8 +4882,15 @@ RJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c</programlisting>
   </appendix>
   <appendix xml:id="_Ref460769599">
     <title>JWT over SCTP example</title>
-    <para>JWT, whose content example is shown in <xref xmlns:xlink="http://www.w3.org/1999/xlink"
-        linkend="q5l_q5m_51c"/>, can be used in the WS-Security header of the SOAP request. To conform to the BinarySecurityToken format, the JWT is base64-encoded a second time.</para>
+    <para>A JWT, whose content example is shown in <xref xmlns:xlink="http://www.w3.org/1999/xlink"
+        linkend="q5l_q5m_51c"/>, can be used in the WS-Security header of the SOAP request. To conform
+        to the BinarySecurityToken format, the JWT itself must be base64-encoded before being embedded
+        in the request:</para>
+    <programlisting><![CDATA[ZXlKMGVYQWlPaUpLVjFRaUxDSmhiR2NpT2lKRlV6STFOaUo5LmV5SnBjM01pT2lKelpYSjJaWEl1Y0dGMGFDNW
+piMjBpTENKdVltWWlPakUyTnpBME9ESTRNREFzSW1WNGNDSTZNVFkzTURVM01qZ3dNQ3dpWVhWa0lqb2lkR0Z5
+WjJWMExtRjFaR2xsYm1ObElpd2ljbTlzWlhNaU9sc2liMjUyYVdZNlQzQmxjbUYwYjNJaVhYMD0uU2ZsS3h3Uk
```

Comment:
@jmelancongen, is this value(i.e ZXlKMGVYQWlPaUpLV...) is equivalent to base64 encoded value of JWT token computed above?

Reply:
It was, with the exception of an extra "=" padding in the middle section that I was supposed to remove. It's updated with exactly the same token as Annex A now.